SOLUTIONS/A new way to say hello world

```diff
@@ -0,0 +1,20 @@
+public class hello
+{
+
+ public static void main(String args[])
+ {
+ 
+ System.out.print("H");
+ System.out.print("E");
+ System.out.print("Y");
+ System.out.print("L");
+ System.out.print("L");
+ System.out.print("O"+\t);
```

Comment:
please add double quotes __"\t"__ around _\t_ on `Line 12` & rename file and add **.java** extension too

Reply:
Thanks @ashwek